### PR TITLE
Update Primitivez and update Components

### PR DESCRIPTION
**Pull request by peduarte**
This PR Updates to the latest version of Primitves and:
- Re-export new components
- Update Checkbox to rely on Primitives
- Update Radio to rely on Primitives
- Remove RadioGroup to rely on Primitives

### CODE REVIEW

File: packages/radix/src/components/Checkbox.tsx [hunk 1]
```diff
@@ -1,99 +1,40 @@
...
+          hover: {
+            borderColor: theme.colors.gray500,
+          },
+          focus: {
+            borderColor: theme.colors.gray600,
+          },
+          checked: {
+            backgroundColor: theme.colors.blue600,
+            borderColor: theme.colors.blue600,
+          },
```
Comment (benoitgrelard) [inline packages/radix/src/components/Checkbox.tsx]: should we add at least a quick disabled state with opacity?

File: packages/radix/src/components/Radio.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            border: '1px solid',
+            borderColor: theme.colors.gray400,
+            backgroundColor: theme.colors.white,
+          },
+          hover: {
+            borderColor: theme.colors.gray500,
+          },
+          focus: {
+            borderColor: theme.colors.blue600,
+          },
```
Comment (benoitgrelard) [inline packages/radix/src/components/Radio.tsx]: should we add at least a quick disabled state with opacity?

File: packages/website/src/docs/checkbox-button.mdx [hunk 3]
```diff
@@ -6,7 +6,7 @@ description: Useful to have a more visual representation of a Checkbox.
 
 ```js live
 <CheckboxButton>
-  <EyeIcon />
+  <CircleIcon />
```
Comment (benoitgrelard) [inline packages/website/src/docs/checkbox-button.mdx]: Did you mean that change?

**Comment by StephenHaney** [comment]
Just curious – would it make more sense to define these IDs as constants to eliminate the possibility of typos or missed updates? Also, they could be exported and used in the prim registration.

**Pull request merged**